packages/app/src/cli/services/flow/extension-to-toml.ts

```diff
@@ -33,6 +33,7 @@ export function buildTomlObject(extension: ExtensionRegistration) {
 
   const localExtensionRepresentation = {
     name: extension.title,
+    handle: extension.uuid,
```

Comment:
@MaxLeduc do we want to make this the folder name too?

Reply:
I think we should, the folder name is already sanitized so it makes sense